src/ui-test/utils/testUtils.ts

```diff
@@ -332,6 +332,17 @@ export async function setCamelCatalogVersion(version: string): Promise<void> {
 	await closeEditor('Settings', true);
 }
 
+/**
+ * Set 'Extra-components' in preferences.
+ * @param components 'Extra components' as string respecting Camel Catalog syntax.
+ */
+export async function setAdditionalComponents(components: string): Promise<void> {
```

Comment:
why we need to do this way? it is not possible to do through UI?

Reply:
no, VS Code additional components are not added through UI :-)